### ISSUE TITLE
Trigger Pa4 from outside 

### ISSUE THREAD

**Issue by finipini** [atc1441/ATC_MiThermometer#202]
Hello. It can possible to some how trigger the reset pad (Pa4) from outside firmware, like tasmota or esphome?

**Comment by atc1441** [comment]
https://github.com/pvvx/ATC_MiThermometer#temperature-or-humidity-trigger-on-gpio-pa5-label-on-the-reset-pin

**Comment by finipini** [comment]
Yes i know ( and i meant pa5) but can i Control that output via an outside trigger? That only trigger within a temperatureor humidity threshold range. 

⁣Obter o BlueMail para Android ​

Em 10/07/2021, 20:53, em 20:53, atc1441 ***@***.***> escreveu:
